congo.go

```diff
@@ -0,0 +1,22 @@
+package congo
+
+// User represents a person in the system.
+// This can be an attendee, speaker, sponsor contact, etc.
+type User struct {
+	ID        uint64
+	FirstName string
+	LastName  string
+	Email     string
+}
+
+// Event represents a specific conference event within a Series.
+type Event struct {
+	ID   uint64
+	Name string
+}
+
+// Ticket represents the admission to an Event for a User.
+type Ticket struct {
+	UserID  uint64
+	EventID uint64
+}
```

Comment:
These are the application types. They're the data types used by other future packages such as an HTTP front end or a API or a CLI. They're agnostic from the underlying storage and is only kept in memory momentarily. The persisted forms of these are in the `store/bolt/internal` package.